src/content/docs/en/pages/devtools/cli/azion-cli-alpha/personal-tokens/personal-tokens.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Azion CLI's personal tokens command and its subcommands
+title: Azion CLI personal-token
 description: Manage your personal tokens through Azion CLI.
 meta_tags: cli, next, javascript, astro, vue, react, edge computing
 namespace: documentation_cli_personal_tokens
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/personal-tokens/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```